.github/workflows/publish.yml

```diff
@@ -0,0 +1,36 @@
+name: Test OSX and Linux
+
+on:
+  create:
+    tags:
+      - v*
+
+jobs:
+  npm:
```

Comment:
I think best would be to focus on the rust stuff and keep this in travis. I'm not aware of anyone actually using this atm and I want you to focus on rust. (I think this is also broken anyway)

Reply:
Yep.